packages/flame/example/lib/main.dart

```diff
@@ -14,14 +14,51 @@ void main() {
   );
 }
 
-class Square extends PositionComponent {
+/// This example simply adds a rotating white square on the screen, if you press
+/// somewhere other than on the existing square another square will be added and
+/// if you press on a square it will be removed.
+class MyGame extends FlameGame with DoubleTapDetector, HasTappables {
+  bool running = true;
+
+  @override
+  Future<void> onLoad() async {
+    await super.onLoad();
+    add(Square(Vector2(100, 200)));
+  }
+
+  @override
+  void onTapUp(int id, TapUpInfo info) {
+    super.onTapUp(id, info);
+    final touchPoint = info.eventPosition.game;
+    final handled = children.any((c) => c.shouldRemove);
```

Comment:
interesting approach!

Reply:
I put a TODO on the onTap* that it should return whether any child handled the tap (returned false)